src/request.js

```diff
@@ -525,6 +525,7 @@ function getDefaultHeaders() {
 
 	return {
 		Depth: '0',
-		'Content-Type': 'application/xml; charset=utf-8'
+		'Content-Type': 'application/xml; charset=utf-8',
+		'X-NC-CalDAV-Webcal-Caching': 'On'
```

Comment:
Can't we set it only for calendar related requests and not as default ?

Reply:
We could, but that would require much overriding plenty of `DavCollection` methods in Calendar. And sending it all the time should not do any harm for now :)

Reply:
@tcitworld Are you fine with this change or should we change it? :)

Reply:
Yup, we'll handle this later.